packages/lesswrong/components/posts/PostsPage/PostBodyPrefix.tsx

```diff
@@ -121,7 +121,7 @@ const PostBodyPrefix = ({post, query, classes}: {
         <Info className={classes.infoIcon}/>
       </LWTooltip>
     </div>}
-    <LinkPostMessage post={post} negativeTopMargin />
+    <LinkPostMessage post={post} />
```

Comment:
I don't know exactly what motivated this change and whether it should be kept (and if, whether it should be forum-gated).  @Discordius ?